### PR TITLE
Cryopods from Oracle Station

### PR DESCRIPTION
**Pull request by Poojawa**
It's really nice to have a /tg/ based station that's porting good shit.

:cl: Poojawa
add: Cryopods are now available for safe round exiting, no longer will you need to ahelp with 'oh fuck wrong job'. Being kicked back to lobby for a restart is still admin. You will be warned to ahelp if you're an antag role however.
/:cl:

These just need to be mapped @deathride58 whenever you get around to it. <3

### CODE REVIEW

**Comment by deathride58** [comment]
Remind me to get to work on a map layering system to make map additions as modular as the rest of our code when the part 1 of combat reworks gets a merge

File: modular_citadel/code/game/machinery/cryopod.dm [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+
+	if(target == user && world.time - target.client.cryo_warned > 5 * 600)//if we haven't warned them in the last 5 minutes
+		var/caught = FALSE
+		if(target.mind.assigned_role in GLOB.command_positions)
+			alert("<span class='userdanger'>You're a Head of Staff![generic_plsnoleave_message]</span>")
+			caught = TRUE
+		if(iscultist(target) || is_servant_of_ratvar(target))
+			to_chat(target, "<span class='userdanger'>You're a Cultist![generic_plsnoleave_message]</span>")
+			caught = TRUE
+		if(istype(SSticker.mode, /datum/antagonist/blob))
```
Comment (deathride58) [inline modular_citadel/code/game/machinery/cryopod.dm:404]: Blob aint a mode anymore, ree!

File: modular_citadel/code/game/machinery/cryopod.dm [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+			if(target.mind in GLOB.overminds)
+				alert("<span class='userdanger'>You're a Blob![generic_plsnoleave_message]</span>")
+				caught = TRUE
+		if(is_devil(target))
+			alert("<span class='userdanger'>You're a Devil![generic_plsnoleave_message]</span>")
+			caught = TRUE
+		if(istype(SSticker.mode, /datum/antagonist/rev))
```
Comment (deathride58) [inline modular_citadel/code/game/machinery/cryopod.dm:411]: Test this to make sure it actually triggers during rev. I don't think SSticker.mode uses antag datums to determine gamemodes.

**Review by deathride58 — CHANGES_REQUESTED** [review]

File: modular_citadel/code/game/machinery/cryopod.dm (hunk 1 above)
Comment (Poojawa) [reply modular_citadel/code/game/machinery/cryopod.dm:404]: sanity just in case.

File: modular_citadel/code/game/machinery/cryopod.dm (hunk 2 above)
Comment (Poojawa) [reply modular_citadel/code/game/machinery/cryopod.dm:411]: It's what I saw most other code using, sooo.

This is also just purely a catch people thing as well, and it worked for cult mode, sooo.

**Comment by deathride58** [comment]
If this breaks shit, you won't be gettin headpats, dork.

**Comment by Poojawa** [comment]
Reeee. Shuttles are shitcode.

**Pull request merged**